rust-toolchain.toml

```diff
@@ -1,5 +1,5 @@
 [toolchain]
-channel = "nightly-2023-10-08"
+channel = "nightly-2024-02-08"
```

Comment:
Why not a more recent build?

Reply:
I decided to take the nearest version that allows to fix security issues as it's primary goal. Otherwise, I need to refactor more code and PR becomes bigger for review. 

I suggest to bump rust version to recent build in separate PR.